multiqc/modules/goleft_indexcov/goleft_indexcov.py

```diff
@@ -0,0 +1,144 @@
+"""MultiQC module to plot output from goleft indexcov
+
+https://github.com/brentp/goleft/tree/master/indexcov
+"""
+from __future__ import print_function
+import collections
+import logging
+
+from multiqc import config
+from multiqc.plots import linegraph, scatter
+from multiqc.modules.base_module import BaseMultiqcModule
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    def __init__(self):
+        super(MultiqcModule, self).__init__(name='goleft indexcov', anchor='goleft_indexcov',
+                                            href='https://github.com/brentp/goleft/tree/master/indexcov',
+                                            info="quickly estimates coverage from a whole-genome bam index.")
+        # Skip coverage plot, too resource intensive with multiple samples
+        # cov_plot = self.coverage_plot()
+        cov_plot = None
+        roc_plot = self.roc_plot()
+        bin_plot = self.bin_plot()
+        if not roc_plot and not bin_plot:
+            log.debug("Did not find goleft indexcov outputs in {}".format(config.analysis_dir))
+            raise UserWarning
+
+        self.sections = list()
+        if cov_plot:
+            self.sections.append({
+                'name': 'Read coverage',
+                'anchor': 'goleft_indexcov-coverage',
+                'content': cov_plot
+            })
+        if roc_plot:
+            self.sections.append({
+                'name': 'Scaled coverage ROC plot',
+                'anchor': 'goleft_indexcov-roc',
+                'content': roc_plot
+            })
+        if bin_plot:
+            self.sections.append({
+                'name': 'Problem coverage bins',
+                'anchor': 'goleft_indexcov-roc',
+                'content': bin_plot
+            })
+
+    def _short_chrom(self, chrom):
```

Comment:
ps. Any reason to use `in set(["X"])` instead of just `in ["X"]` out of interest?